src/table/src/requests.rs

```diff
@@ -53,7 +53,6 @@ pub fn validate_table_option(key: &str) -> bool {
         // common keys:
         WRITE_BUFFER_SIZE_KEY,
         TTL_KEY,
-        REGIONS_KEY,
```

Comment:
This change make `regions = 1` an illegal option and thus a breaking change.

@killme2008 @waynexia please confirm if we should allow `regions = 1` but leave it unused, or we can break and throw an `InvalidTableOption` error.

Reply:
The `regions` option is for future usage, and we must support multiple regions for standalone mode.

Reply:
I think we can just ignore it currently but doesn't need to break it.

Reply:
OK. Then I'll add back this option in validation so such an option can be passed but with no effect.

Reply:
No hurry, looks like we don't need it at all even in standalone mode.